Project/Assets/Prairie/Framework/Script/Interaction/FirstPersonInteractor.cs

```diff
@@ -82,14 +85,19 @@ void OnGUI()
 		else
```

Comment:
Response: Good call! A _blank_ string should totally be treated the same.

Joke: I'll also make it so if you type "plz dont show this prompt" it is hidden.